db/init_db.js

```diff
@@ -9,8 +10,51 @@ async function buildTables() {
     client.connect();
 
     // drop tables in correct order
+    console.log("Dropping tables...")
+    await client.query(`
+      DROP TABLE IF EXISTS order_products;
+      DROP TABLE IF EXISTS orders;
+      DROP TABLE IF EXISTS products;
+      DROP TABLE IF EXISTS users;
+    `)
+
 
     // build tables in correct order
+    console.log("Creating tables...")
+    await client.query(`
+      CREATE TABLE products(
+        id SERIAL PRIMARY KEY,
+        name VARCHAR(255) NOT NULL,
+        description TEXT NOT NULL,
+        price INTEGER NOT NULL,
+        "imageURL" VARCHAR(255) DEFAULT 'https://placeimg.com/300/300/any',
+        "inStock" BOOLEAN DEFAULT false,
+        category VARCHAR(255) NOT NULL
+      );
+      CREATE TABLE users(
+        id SERIAL PRIMARY KEY,
+        "firstName" VARCHAR(255) NOT NULL,
+        "lastName" VARCHAR(255) NOT NULL,
+        email VARCHAR(255) UNIQUE NOT NULL,
+        "imageURL" VARCHAR(255) DEFAULT 'https://placeimg.com/100/100/people',
+        username VARCHAR(255) UNIQUE NOT NULL,
+        password VARCHAR(255) UNIQUE NOT NULL,
+        "isAdmin" BOOLEAN UNIQUE NOT NULL DEFAULT false
+      );
+      CREATE TABLE orders(
+        id SERIAL PRIMARY KEY,
+        status VARCHAR(255) DEFAULT 'created',
+        "userId" INTEGER REFERENCES users(id),
+        "datePlaced" DATE
+      );
+      CREATE TABLE order_products(
+        id SERIAL PRIMARY KEY,
+        "productId" INTEGER REFERENCES products(id),
+        "orderId" INTEGER REFERENCES orders(id),
+        price INTEGER NOT NULL,
```

Comment:
Maybe change to float, if changing the above?

Reply:
See above.